ui/app/src/components/ResizeableDrawer/ResizeableDrawer.tsx

```diff
@@ -95,13 +97,13 @@ const useStyles = makeStyles<ResizeableDrawerStyles, ResizeableDrawerClassKey>()
       ...drawerPaperRight
     },
     resizeHandle: {
-      width: '1px',
+      width: '4px',
```

Comment:
4px is the width of the handle when active, so by setting this at the same width we make it easier to grab with no visual difference (since this is not visible when inactive).